Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazeInput.cpp

```diff
@@ -109,6 +116,7 @@ DependencyProperty^ GazeInput::RepeatDelayDurationProperty::get() { return s_rep
 DependencyProperty^ GazeInput::DwellRepeatDurationProperty::get() { return s_dwellRepeatDurationProperty; }
 DependencyProperty^ GazeInput::ThresholdDurationProperty::get() { return s_thresholdDurationProperty; }
 DependencyProperty^ GazeInput::MaxDwellRepeatCountProperty::get() { return s_maxRepeatCountProperty; }
+DependencyProperty^ GazeInput::IsSwitchEnabled::get() { return s_isSwitchEnabled; }
```

Comment:
Perhaps rename IsSwitchEnabled and s_isSwitchEnabled to IsSwichEnabledProperty and s_isSwitchEnabledProperty. There are conventions expected and sometimes enforced around the naming of these things.

Reply:
Sure thing - good catch.